### DISCUSSION TITLE
Error loading pymdownx.extra to enable Html+Markdown

### DISCUSSION THREAD

**Discussion by vivekragunathan** [mkdocs/mkdocs#2657] [General]
I can't for the life of me figure out how to get `pymdownx.extra` working.

Based on what I read, I need to enable this in the `mkdocs.yaml` so that I can have html + markdown. For instance ...

```
<div>This is **bold** and beautiful</div>


The markdown inside div should be treated as markdown, and the word bold should be rendered bold.
```

`pymdownx.extra` seems to be the one that can help with that. But adding `pymdownx.extra` under `markdown_extensions` (or other places say `plugins`) gives an error and aborts loading.

```
➜ mkdocs serve
INFO     -  Building documentation...
ERROR    -  Config value: 'markdown_extensions'. Error: Failed loading extension "pymdownx.extra".
Aborted with 1 Configuration Errors!
```

I have installed `pymdown-extensions` on my machine (`pip3 install pymdown-extensions`).

I am using mkdocs for the first time. Help appreciated.

**Comment by oprypin** [comment]
Could you try this method of installation, to make sure there aren't separate disjointed installations of Python:

```shell
python3 -m pip install mkdocs pymdown-extensions
python3 -m mkdocs serve
```
